test/utils.go

```diff
@@ -19,3 +36,115 @@ func DBTestCredentials() database.DBCredentials {
 		Port:     "17017",
 	}
 }
+
+// CreateUser is a helper method which simplifies the creation of test users
+func CreateUser(t *testing.T, at *AccountsTester, customEmail, customPassword string) (*database.User, func(user *database.User), error) {
+	email := customEmail
+	if email == "" {
+		// Use the test's name as an email-compatible identifier.
+		email = strings.ReplaceAll(t.Name(), "/", "_") + "@siasky.net"
+	}
+	password := customPassword
+	if password == "" {
+		password = hex.EncodeToString(fastrand.Bytes(16))
+	}
+	params := map[string]string{
+		"email":    email,
+		"password": password,
+	}
+	// Create a user.
+	_, _, err := at.Post("/user", nil, params)
+	if err != nil {
+		return nil, nil, errors.AddContext(err, "user creation failed")
+	}
+	// Fetch the user from the DB, so we can delete it later.
+	u, err := at.DB.UserByEmail(at.Ctx, email, false)
+	if err != nil {
+		return nil, nil, errors.AddContext(err, "failed to fetch user from the DB")
+	}
+	cleanup := func(user *database.User) {
```

Comment:
Old habits die hard, I guess.

This approach resolves the issue that forced me to pass the `*testing.T` param in https://github.com/SkynetLabs/skynet-accounts/pull/78#discussion_r721423178, so that's great.